content/en/platform/corda/5.1/application-networks/managing/endpoints/_index.md

```diff
@@ -12,7 +12,7 @@ section_menu: corda51
 ---
 # Member Endpoints
```

Comment:
@YashNabar should member endpoints section be placed where it is now (under Managing Members): https://anna.preview.docs.r3.com/en/platform/corda/5.1/application-networks/managing.html
OR should I move it under Onboarding Members here: https://anna.preview.docs.r3.com/en/platform/corda/5.1/application-networks/creating/members.html

Reply:
@AnnaK-R3 You're right, I think it should be under Onboarding Members

Reply:
Moved